layouts/base.njk

```diff
@@ -71,6 +70,6 @@
 
 {% block bodyEnd %}
   <script src="{{ '/assets/govuk.js' | url }}"></script>
-  <script>document.documentElement.classList.remove('no-js')</script>
+  <script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
```

Comment:
I don't think this is needed, as it's already included within `"govuk/template.njk"`, which adds it immediately after `<body>`? At the moment it's appearing twice.

```suggestion
```

Reply:
Ah, looks like this is relied upon by our site search component, which expects a `no-js` to be on the body, and then removed by this script. Need to dig into this a bit more.